e2e/scenario_test/settings/not_set/delivery_limits/settings.no_delivery_limits.spec.js

```diff
@@ -0,0 +1,50 @@
+const { setup, pump, cgm, match, settings } = require('../../../../../src/index');
+
+describe('Pump Settings', () => {
+    beforeAll(async () => {
+        await setup.LaunchLoop();
+        await setup.LoadScenarios(device.id);
+        await cgm.Add();
+        await pump.Add();
+    });
+    describe('Closed loop is not allowed', () => {
+        describe('When delivery limits are not set', () => {
+            beforeAll(async () => {
+                await setup.LoadScenario('flat_cgm');
+            });
+            it('should NOT set the suspend threshold', async () => {
+                await settings.Suspend('75');
+            });
+            it('should have correction range set', async () => {
+                await settings.CorrectionRanges([{time: '12:00 AM', min:'179',max:'180'}]);
+            });
+            it('should set the basal rates', async () => {
+                await settings.BasalRates('0.1')
+            });
+            it.skip('should not set the delivery limits', async () => {
+                await settings.DeliveryLimits('', '');
```

Comment:
[nit] Perhaps just remove this code and leave a comment as to why it is missing for this test? (Rather than setting `('', '')` which is unclear as to intent.)